tsconfig.build.json

```diff
@@ -1,8 +1,10 @@
 {
   "references": [
-    { "path": "./packages/design-system-react/tsconfig.build.json" },
+    { "path": "./packages/design-tokens/tsconfig.build.json" },
+    { "path": "./packages/design-system-twrnc-preset/tsconfig.build.json" },
     { "path": "./packages/design-system-tailwind-preset/tsconfig.build.json" },
-    { "path": "./packages/design-tokens/tsconfig.build.json" }
+    { "path": "./packages/design-system-react-native/tsconfig.build.json" },
+    { "path": "./packages/design-system-react/tsconfig.build.json" }
```

Comment:
Add React Native packages to TypeScript project references

Restores React Native package references in the root tsconfig.build.json to ensure proper TypeScript type checking and build ordering. While the build process now uses Yarn workspaces for dependency management, maintaining TypeScript project references is still valuable because:

1. It provides accurate TypeScript IntelliSense and type checking across packages during development
2. Enables proper handling of type declarations when packages import from each other
3. Ensures consistent type checking behavior regardless of build tool

This complements the Yarn workspace build process while maintaining strong TypeScript integration across the monorepo.